.github/workflows/release.yml

```diff
@@ -368,15 +368,15 @@ jobs:
           profile: ${{matrix.build-profile}}
           package: ${{matrix.package}}
           chain: ${{matrix.chain}}
-          tag: "1.84.1"
+          tag: "1.88.0"
           runtime_dir: ${{ matrix.runtime-dir }}
       - name: Rename WASM file
         run: |
           cp -p ${{ steps.srtool_build.outputs.wasm_compressed }} \
             ./${{env.WASM_DIR}}/${{env.RELEASE_WASM_FILENAME}}
       - name: Install Subwasm
         run: |
-          cargo install --locked --git https://github.com/chevdor/subwasm --tag v0.21.3 --force
+          cargo install --git https://github.com/chevdor/subwasm  --force
```

Comment:
nit: do we need to lock this to a specific tag or commit just to reduce potential supply/chain attacks or unexpected changes?